flake: `schemacaching/standardcaching_test.go`

https://github.com/authzed/spicedb/actions/runs/8020113908/job/21909300689?pr=1757
```
panic: 
assert: mock: The method has been called over 1 times.
	Either do one more Mock.On("ReadNamespaceByName").Return(...), or remove extra call.
	This call was unexpected:
		ReadNamespaceByName(string)
		0: "namespace_a"
	at: [/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/proxy_test/mock.go:91 /home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:66 /home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:177 /home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:198 /home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:200 /home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:113 /home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:174 /home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:64 /home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcaching_test.go:79 /home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcaching_test.go:310 /home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/errgroup/errgroup.go:78 /opt/hostedtoolcache/go/1.21.7/x64/src/runtime/asm_amd64.s:1650]

runtime/debug.Stack()
	/opt/hostedtoolcache/go/1.21.7/x64/src/runtime/debug/stack.go:24 +0x67
golang.org/x/sync/singleflight.newPanicError({0x14c3460?, 0xc0001cea70})
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:44 +0x30
golang.org/x/sync/singleflight.(*Group).doCall.func2.1()
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:193 +0x56
panic({0x14c3460?, 0xc0001cea70?})
	/opt/hostedtoolcache/go/1.21.7/x64/src/runtime/panic.go:920 +0x270
github.com/stretchr/testify/mock.(*Mock).fail(0xc00001cd70, {0x16dd6c3, 0xa5}, {0xc00028a7c0, 0x4, 0x4})
	/home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.4/mock/mock.go:3[32](https://github.com/authzed/spicedb/actions/runs/8020113908/job/21909300689?pr=1757#step:4:33) +0x1d9
github.com/stretchr/testify/mock.(*Mock).MethodCalled(0xc00001cd70, {0x1b49884, 0x13}, {0xc0001ce7e0?, 0x1, 0x1})
	/home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.4/mock/mock.go:480 +0x352
github.com/stretchr/testify/mock.(*Mock).Called(0x16a84cb?, {0xc0001ce7e0, 0x1, 0x1})
	/home/runner/go/pkg/mod/github.com/stretchr/testify@v1.8.4/mock/mock.go:464 +0x193
github.com/authzed/spicedb/internal/datastore/proxy/proxy_test.(*MockReader).ReadNamespaceByName(0xc00001cd70, {0x0?, 0x0?}, {0x16a84cb, 0xb})
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/proxy_test/mock.go:91 +0xc5
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.(*definitionCachingReader).ReadNamespaceByName.func1({0x18e4b70, 0xc0004d80c0}, {0x16a84cb, 0xb})
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:66 +0x74
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.readAndCache[...].func1()
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:177 +0xb6
golang.org/x/sync/singleflight.(*Group).doCall.func2(0xc0008c7b17, 0xc0000e23c0, 0xc0008c7d38)
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:198 +0xc4
golang.org/x/sync/singleflight.(*Group).doCall(0xc0004d8e90, 0xc0000e23c0, {0xc0005be2c1, 0xf}, 0x0?)
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:200 +0x154
golang.org/x/sync/singleflight.(*Group).Do(0xc0004d8e90, {0xc0005be2c1, 0xf}, 0x18d25f8?)
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:113 +0x205
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.readAndCache[...]({0x18e4a78, 0x21bd780?}, 0xc0004d8090, {0x18d25f8, 0x1}, {0x16a84cb, 0xb}, 0xc0008c7e18, 0x16ff538)
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:174 +0x2f1
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.(*definitionCachingReader).ReadNamespaceByName(0xc0004d8090, {0x18e4a78, 0x21bd780}, {0x16a84cb, 0xb})
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:64 +0xd1
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.glob..func1({0x18e4a78, 0x21bd780}, {0x18ea2a8, 0xc0004d8090}, {0x16a84cb, 0xb})
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcaching_test.go:79 +0x72
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.TestSingleFlight.func1.1()
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcaching_test.go:310 +0xde
golang.org/x/sync/errgroup.(*Group).Go.func1()
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/errgroup/errgroup.go:78 +0x98
created by golang.org/x/sync/errgroup.(*Group).Go in goroutine 359
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/errgroup/errgroup.go:75 +0x125


goroutine 361 [running]:
golang.org/x/sync/singleflight.(*Group).doCall.func1()
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:170 +0x5[33](https://github.com/authzed/spicedb/actions/runs/8020113908/job/21909300689?pr=1757#step:4:34)
golang.org/x/sync/singleflight.(*Group).doCall(0xc0004d8e90, 0xc0000e23c0, {0xc0005be2c1, 0xf}, 0x0?)
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:205 +0x165
golang.org/x/sync/singleflight.(*Group).Do(0xc0004d8e90, {0xc0005be2c1, 0xf}, 0x18d25f8?)
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/singleflight/singleflight.go:113 +0x205
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.readAndCache[...]({0x18e4a78, 0x21bd780?}, 0xc0004d8090, {0x18d25f8, 0x1}, {0x16a84cb, 0xb}, 0xc0008c7e18, 0x16ff538)
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:174 +0x2f1
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.(*definitionCachingReader).ReadNamespaceByName(0xc0004d8090, {0x18e4a78, 0x21bd780}, {0x16a84cb, 0xb})
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcache.go:64 +0xd1
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.glob..func1({0x18e4a78, 0x21bd780}, {0x18ea2a8, 0xc0004d8090}, {0x16a84cb, 0xb})
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcaching_test.go:79 +0x72
github.com/authzed/spicedb/internal/datastore/proxy/schemacaching.TestSingleFlight.func1.1()
	/home/runner/actions-runner/_work/spicedb/spicedb/internal/datastore/proxy/schemacaching/standardcaching_test.go:310 +0xde
golang.org/x/sync/errgroup.(*Group).Go.func1()
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/errgroup/errgroup.go:78 +0x98
created by golang.org/x/sync/errgroup.(*Group).Go in goroutine [35](https://github.com/authzed/spicedb/actions/runs/8020113908/job/21909300689?pr=1757#step:4:36)9
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.6.0/errgroup/errgroup.go:[75](https://github.com/authzed/spicedb/actions/runs/8020113908/job/21909300689?pr=1757#step:4:76) +0x125
```

--- a/internal/datastore/proxy/schemacaching/standardcaching_test.go
+++ b/internal/datastore/proxy/schemacaching/standardcaching_test.go
@@ -298,7 +298,7 @@ func TestSingleFlight(t *testing.T) {
 			dsMock.On("SnapshotReader", one).Return(oneReader)
 			oneReader.
 				On(tester.readSingleFunctionName, nsA).
-				WaitUntil(time.After(10*time.Millisecond)).
+				WaitUntil(time.After(50*time.Millisecond)).
 				Return(nil, old, nil).
 				Once()
 

